rerun_py/rerun_sdk/rerun/archetypes/arrows3d_ext.py

```diff
@@ -47,12 +48,23 @@ def __init__(
 
         # Custom constructor to remove positional arguments and force use of keyword arguments
         # while still making vectors required.
+        with catch_and_log_exceptions(context=self.__class__.__name__):
+            self.__attrs_init__(
+                vectors=vectors,
+                origins=origins,
+                radii=radii,
+                colors=colors,
+                labels=labels,
+                class_ids=class_ids,
+                instance_keys=instance_keys,
+            )
+            return
         self.__attrs_init__(
-            vectors=vectors,
-            origins=origins,
-            radii=radii,
-            colors=colors,
-            labels=labels,
-            class_ids=class_ids,
-            instance_keys=instance_keys,
+            vectors=None,
+            origins=None,
+            radii=None,
+            colors=None,
+            labels=None,
+            class_ids=None,
+            instance_keys=None,
```

Comment:
does one really have to spell out all the components here? That seems error prone, especially when adding new components.

Cannot we call the auto-generate a `Arrows3_clear()` method here instead?

Reply:
Yeah, this was annoying me too -- I think I have an idea for a better pattern.

Reply:
Ok, this is now replaced by a generated `__attrs_clear__()`